app/assets/stylesheets/master.scss

```diff
@@ -1 +1,19 @@
 @import "bootstrap";
+
```

Comment:
@rpolacheck Looks like we have duplication here. I put the .white and .black classes in games.scss, but you also put them in master.scss, slightly differently. @mscoutermarsh which file would make more sense for us to put these classes in?